apps/test/unit/code-studio/initApp/projectTest.js

```diff
@@ -180,6 +180,7 @@ describe('project.js', () => {
     });
 
     it('default case', () => {
+      window.appOptions.app = 'someOtherType';
```

Comment:
Why do you have this line twice?

Reply:
😅 I have no idea how that got there, removed it!